### PR TITLE
[feat] Added Grids component

### PR DESCRIPTION
**Pull request by omjadhav85**
Added Grids component

P.S. Files to be reviewed are only "routes/grids.html" and "styles/components/grids.css". Other files are just inclusion of new anchor tag for Grids page in other pages.

image
image

### CODE REVIEW

File: routes/grids.html [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        <ul class="nav-actions">
+          <li class="nav-action-item"><a href="../index.html">Home</a></li>
+          <li class="nav-action-item"><a href="../index.html">Docs</a></li>
+          <li class="nav-action-item">
+            <a href="https://github.com/bLAckBLAZER/Component-Library"
+              >Source</a
+            >
+          </li>
+        </ul>
+      </nav>
```
Comment (junaidshaikh-js) [inline routes/grids.html:28]: Can be nested inside the `header` landmark element.

**Review by junaidshaikh-js — APPROVED** [review]
Good work done! 

Many grid examples are added. You can change the HTML as suggested in the comment.

Can be merged.

**Review by abhij1607 — COMMENTED** [review]
Very Clean code | | Easy to read | | Good variable naming

Apart from Junaid suggestion, everything seems good

File: routes/grids.html (hunk 1 above)
Comment (omjadhav85) [reply routes/grids.html:28]: Hey! I had tried putting nav in header tag but it was causing issues with position: sticky. So decided to take it out.

**Comment by omjadhav85** [comment]
Thanks Abhi and Junaid. Merging after 2 reviews.

**Pull request merged**